doc/source/drivers/vector/mssqlspatial.rst

```diff
@@ -208,3 +214,11 @@ Connecting with username/password
    ::
    
       ogrinfo -al   MSSQL:server=.\MSSQLSERVER2008;database=geodb;trusted_connection=no;UID=user;PWD=pwd
+
+Connecting with username/password stored in environment variables
+
+   ::
+   
+      export MSSQLSPATIAL_UID=user
+      export MSSQLSPATIAL_PWD=pwd
+      ogrinfo -al   MSSQL:server=.\MSSQLSERVER2008;database=geodb;trusted_connection=no
```

Comment:
is is trusted_connection or trustedconnection ? There's a mix of boths in your PR

Reply:
Thanks for pointing that out. It should be _trusted_connection_. Will double check and fix accordingly.